README.md

```diff
@@ -9,10 +9,36 @@ Created ssh-key, checked the environment, saw __OhMyZsh__ been updated with pic
 
 <img src="Image/OMZ.png" width="500" >
 
-which is inspiring for me to __keep going__. :smile:
+which inspire me to __keep going__. :smile:
 
 I edit code in Atom, type command line in terminal, learn how to use MARKDOWN to edit README.
 
 Finally, I know how to update file on Github through git config.
 
 Thanks my mentor @jerome1210
+___
+
+### 02Feb.2023- Learning Note
+**javaScript**
+- Callback
+```javaScript
+#example
+function shout(){
+  console.log("Help!");
+};
+function countToThree(callback){
+  for(var i = 0; i <= 3; i++){
+    console.log(i);
+  };
+  callback();
+};
+countToThree(shout);
+```
```

Comment:
I would prefer to log this into a directory like excism and leetcode to record your learning and use the link in the readme note.